air/src/stack/mod.rs

```diff
@@ -23,6 +24,20 @@ const B1_COL_IDX: usize = B0_COL_IDX + 1;
 /// inputs and outputs as well as the initial values of the bookkeeping columns.
 pub const NUM_ASSERTIONS: usize = 2 * MIN_STACK_DEPTH + 2;
 
+/// The number of generic constraints in the stack operations.
+pub const NUM_GENERIC_CONSTRAINTS: usize = 16;
```

Comment:
nit: I'm wonder if the word `generic` is a good one to use here. "generic" has a special meaning in Rust and other languages, maybe a different word would be better - e.g., "general" or maybe "common" or something like that.